nat-lab/tests/test_events_link_state.py

```diff
@@ -213,5 +213,7 @@ async def test_event_link_state_feature_disabled(
 
         # There should be 2 node events Connecting and Connected
```

Comment:
This comment contradicts the code below a bit. It might be useful to paint the full picture - what we expect and why or remove the comment altogether in my opinion :/

Reply:
Removed it entirely